package/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@aurory/lockersdk",
```

```suggestion
  "name": "@aurory/locker-sdk",
```
if we want to adjust the naming / worth creating a new package